src/scripts/postclone.js

```diff
@@ -26,25 +37,87 @@ var class_name,
+    demoTsSearchTerm = ".bindingContext =",
+    demoAngularSearchTerm = 'templateUrl: "app.component.html"',
+    demoVueSearchTerm = '<Page class="page">',
+    preDefinedInclude = [
+        "../src",
+        "**/*"
+    ],
+    preDefinedExclude = [
+        "../src/node_modules",
+        "node_modules",
+        "platforms"
+    ],
+    preDefinedPaths = [
+        {
+            key: "*",
+            value: [
+                "./node_modules/*"
+            ]
+        },
+        {
+            key: "~/*",
+            value: [
+                "app/*"
+            ]
+        }
+    ],
+    appNamePlaceholderStr = "appNamePlaceholder",
+    pluginNamePlaceholderStr = "pluginNamePlaceholder",
+    appPathPlaceholderStr = "appPathPlaceholder",
+    removeAddPluginCommand = "cd appNamePlaceholder && tns plugin remove pluginNamePlaceholder && tns plugin add ../src",
+    removeAddPluginCommandPlaceholderStr = "removeAddPluginCommandPlaceholder",
+    cleanAppsScriptPlaceholderStr = "cleanAppsScriptPlaceholder",
+    preDefinedAppScripts = [
+        {
+            key: "appNamePlaceholder.ios",
+            value: "npm run tsc && cd appPathPlaceholder && tns run ios --syncAllFiles --emulator"
+        },
+        {
+            key: "appNamePlaceholder.android",
+            value: "npm run tsc && cd appPathPlaceholder && tns run android --syncAllFiles --emulator"
+        }],
+    preDefinedPrepareScript =
+    {
+        key: "plugin.prepare",
+        value: "npm run build removeAddPluginCommandPlaceholder"
+    },
+    preDefinedResetScript = {
+        key: "appNamePlaceholder.reset",
+        value: "cd appPathPlaceholder && npx rimraf -- hooks node_modules platforms package-lock.json"
+    },
+    preDefinedCleanScript = {
+        key: "clean",
+        value: "cleanAppsScriptPlaceholder && npx rimraf -- node_modules package-lock.json && npm i",
     };
 
 console.log('NativeScript Plugin Seed Configuration');
 
+// Expected order: `gitHubUsername` `pluginName` `initGit` `includeTypescriptDemo` `includeAngularDemo`
+// Example: gitHubUsername=PluginAuthor pluginName=myPluginClassName initGit=n includeTypescriptDemo=y includeAngularDemo=n
 var parseArgv = function () {
     var argv = Array.prototype.slice.call(process.argv, 2);
     var result = {};
     argv.forEach(function (pairString) {
         var pair = pairString.split('=');
         result[pair[0]] = pair[1];
     });
+
     return result;
 };
 var argv = parseArgv();
 
-if (argv.gitHubUsername !== undefined && argv.pluginName !== undefined && argv.initGit !== undefined) {
+if (argv) {
     inputParams.github_username = argv.gitHubUsername;
     inputParams.plugin_name = argv.pluginName;
-    inputParams.init_git = argv.initGit
+    inputParams.init_git = argv.initGit;
+    inputParams.include_typescript_demo = argv.includeTypescriptDemo;
+    inputParams.include_angular_demo = argv.includeAngularDemo;
+    // inputParams.include_vue_demo = argv.includeVueDemo;
```

Comment:
let's enable the creation of the demo-vue app with a notification and link to the readme that some manual steps are required

Reply:
At the moment we do not know exactly what those manual steps are due to issues in the CLI with webpack + linked TypeScript plugin + Vue app with .vue files. [The issue](https://github.com/NativeScript/nativescript-dev-webpack/issues/823) is still open and some other plugin creators are hitting it.

My suggestion is to not enable it until we have a clearer picture how to handle Vue demo with linked TypeScript plugin.

Reply:
👍